app/helpers/sessions_helper.rb

```diff
@@ -3,13 +3,35 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
+  def forget user
+    user.forget
+    cookies.delete :user_id
+    cookies.delete :remember_token
+  end
+
   def log_out
+    forget current_user
     session.delete :user_id
     @current_user = nil
   end
 
   def current_user
-    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+    if user_id = session[:user_id]
```

Comment:
chỗ này là phép gán, em dùng `()` nhé
```suggestion
    if (user_id = session[:user_id])
```